k8s-samples/existing-lighthouse/add-lighthouse-existing-cluster.md

```diff
@@ -1,39 +1,49 @@
-## Add Lighthouse to an existing portworx cluster
+---
+layout: page
+title: "Add Lighthouse to existing Portworx cluster"
+keywords: portworx, container, Kubernetes, gui, docker, openshift, lighthouse
+meta-description: "Find out how to deploy lighthouse on a cluster where Portworx is already installed"
+---
 
-### Download and install the lighthouse
-
-Step 1: Download the [lighthouse yaml](https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/k8-lighthouse.yaml):
+### Download and install the Lighthouse
 
+##### Step 1: Download the Lighthouse spec file
 ```
 wget https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/k8-lighthouse.yaml
 ```
+For OpenShift,
+```
+wget https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/openshift/k8-lighthouse.yaml
+```
```

Comment:
[usability suggestion]
You might want to leave hyperlinks for YAMLs, so people can check the content just by clicking -- e.g.

```
Download [generic Lighthouse spec file](https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/k8-lighthouse.yaml):

    wget https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/k8-lighthouse.yaml

Download [OpenShift-specific Lighthouse spec file](https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/openshift/k8-lighthouse.yaml):

    wget https://raw.githubusercontent.com/portworx/px-docs/gh-pages/k8s-samples/existing-lighthouse/openshift/k8-lighthouse.yaml
```